docs/docs/basic-features/managing-segments.md

```diff
@@ -118,7 +114,31 @@ contains the tenancy ID of that user.
 
 The `In` operator _is_ case sensitive when evaluating alphabetical characters.
 
-### SemVer-aware operators
+:::important
+
+Earlier SDK versions will not work in local evaluation mode if your environment has segments with the `In` operator
+defined.
+
+To keep local evaluation from breaking, please ensure you have your SDK versions updated before you add such segments to
+your environment.
+
+:::
```

Comment:
It's kinda odd that we have this alert here but not in the Modulo tab. We should probably standardise this in some way.

Reply:
OK added to the modulo and merged